vault/acl.go

```diff
@@ -241,6 +263,24 @@ CHECK:
 		return false, sudo
 	}
 
+	if permissions.MaxWrappingTTL > 0 {
+		if req.WrapInfo == nil || req.WrapInfo.TTL > permissions.MaxWrappingTTL {
```

Comment:
It does! If it's set but WrapInfo is nil, the client didn't request wrapping, and thus we return false.